runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -2001,6 +2002,11 @@ static void jitHookAnonClassesUnload(J9HookInterface * * hookInterface, UDATA ev
    for (J9Class* j9clazz = unloadedEvent->anonymousClassesToUnload; j9clazz; j9clazz = j9clazz->gcLink)
       {
       numClasses++;
+
+#if JAVA_SPEC_VERSION >= 22
+      vmThread->javaVM->internalVMFunctions->hashClassTableMHProxyPackageDelete(vmThread, j9clazz->classLoader, j9clazz->romClass);
```

Comment:
is this onl added to the jit unloadhook, what happens if we run -Xint

Reply:
I will add this to a new vm method when J9HOOK_VM_ANON_CLASSES_UNLOAD is triggered.

Reply:
I made this change and am testing it on aix.